tests/monotouch-test/Foundation/CalendarTest.cs

```diff
@@ -18,6 +18,10 @@
 using nuint=global::System.UInt32;
 #endif
 
+#if MONOMAC
```

Comment:
Should this be PlatformException and defined on MONOTOUCH and MONOMAC? https://github.com/xamarin/xamarin-macios/blob/master/tools/common/Application.cs#L15